report/renderer/category-renderer.js

```diff
@@ -571,7 +572,11 @@ export class CategoryRenderer {
       }
 
       const description = clumpId === 'manual' ? category.manualDescription : undefined;
-      const clumpElem = this.renderClump(clumpId, {auditRefs, description});
+      const scoreOutOf100 = Math.round(Number(category.score) * 100);
```

Comment:
This should work for our use case, but is it possible to make this check for "no audits failing" instead?

Reply:
Should be able to check `clumps.get('failed')?.length` for this.